.github/workflows/python-ci-single.yml

```diff
@@ -103,20 +103,29 @@ jobs:
     - name: Show XCode version
       run: clang --version
 
-    - name: Generate test data
+    - name: Obtain test data
       shell: bash
       run: |
-        mkdir -p test/soco
-        ./apis/python/devtools/ingestor --debug --soco -o test/soco -n data/pbmc3k_processed.h5ad data/10x-pbmc-multiome-v1.0/subset_100_100.h5ad
+        rm -rf test/soco
+        mkdir -p test
+        cd test
+        tar zxf soco.tgz
```

Comment:
cd'ing into a director you just made and then expanding a file you expect in the directory you just made may not work, or I am behind on what types of wheaties I shoud have for breakfast.

Reply:
@eddelbuettel it does indeed work (witness CI passes)

The `mkdir -p` is unnecessary as `test/soco.tgz` is under source control

Reply:
I see, so `mkdir -p test` tricked me as it is "empty" as `test/soco.tgz` exists at the same time.  Still reads weird.

But that is not real worry then.  Thanks for the cluebat.